LFS171x/docs/introduction-to-hyperledger.md

```diff
@@ -1 +1,386 @@
+
+As a Hyperledger module, *"Cello aims to bring the on-demand 'as-a-service' deployment model to the blockchain ecosystem"*, thus helping in furthering the development and deployment of Hyperledger's frameworks. Hyperledger Cello was initially contributed by IBM, with sponsors from Soramitsu, Huawei, and Intel.
+
+Application developers and system administrators using Cello can provision and maintain Hyperledger networks. For instance, you can create a group of distributed ledger networks in virtual clouds known as 'container clusters', and then, manage and monitor those networks with a configurable dashboard. Additionally, you can build a Blockchain-as-a-Service (BaaS) platform.
+
+![Hyperledger Cello](../images/introduction-to-hyperledger/cello.jpg)
+
+Hyperledger Cello (Source: https://www.hyperledger.org/blog/2017/01/17/hyperledger-says-hello-to-cello)
+
+## Hyperledger Explorer
+[Hyperledger Explorer](https://www.hyperledger.org/projects/explorer) is a tool for visualizing blockchain operations. It is the first ever blockchain explorer for permissioned ledgers, allowing anyone to explore the distributed ledger projects being created by Hyperledger's members from the inside, without compromising their privacy. The project was contributed by DTCC, Intel, and IBM.
+
+Designed to create a user-friendly web application, Hyperledger Explorer can view, invoke, deploy, or query:
+
+* Blocks
+* Transactions and associated data
+* Network information (name, status, list of nodes)
+* Smart contracts (chain codes and transaction families)
+* Other relevant information stored in the ledger.
+
+The ability to visualize data is of critical importance, in order to extract business value from it. Hyperledger Explorer provides this much needed functionality. Key components include a web server, a web UI, web sockets, a database, a security repository, and a blockchain implementation.
+
+## Hyperledger Composer
+[Hyperledger Composer](https://www.hyperledger.org/projects/composer) provides a suite of tools for building blockchain business networks. These tools allow you to:
+
+* Model your business blockchain network
+* Generate REST APIs for interacting with your blockchain network
+* Generate a skeleton Angular application.
+
+Built in Javascript, Hyperledger Composer provides an easy-to-use set of components that developers can quickly learn and implement. The project was contributed by Oxchains and IBM.
+
+## Hyperledger Composer (Simon Stone & Kathryn Harrison)
+[![Hyperledger Composer (Simon Stone & Kathryn Harrison)](../images/video-image.png)](https://youtu.be/91qtaojB9oI)
+[Some rights reserved](https://creativecommons.org/licenses/by-nc-nd/4.0/)
```

Comment:
Remove - the course is actually licensed under https://creativecommons.org/licenses/by/4.0/. Unfortunately, EdX doesn't let you specify which Creative Common license for videos. Therefore, we will let the overall license cover these.